kubernetes/README.md

```diff
@@ -23,6 +23,27 @@ There are some yaml files for deploying apisix in Kubernetes.
 ### Prerequisites
 - Install etcd
 
+#### when using etcd-operator
+when using etcd-operator, you need to change apisix-gw-config-cm.yaml:
+
+* add CoreDNS IP into dns_resolver
+
+```
+dns_resolver:
+  - 10.233.0.3      # default coreDNS cluster ip
```

Comment:
We should comment this `dns_resolver`, as APISIX get the exact dns ip from pod's `/etc/resolv.conf`

Reply:
As far as i test, apisix/nginx do not use `/etc/resolv.conf` by default.
maybe another way is that let nginx read  `/etc/resolv.conf`.

Reply:
> As far as i test, apisix/nginx do not use `/etc/resolv.conf` by default.
> maybe another way is that let nginx read `/etc/resolv.conf`.

Apache APISIX will read from `/etc/resolv.conf`, if not, it will be a bug.

Reply:
> > As far as i test, apisix/nginx do not use `/etc/resolv.conf` by default.
> > maybe another way is that let nginx read `/etc/resolv.conf`.
> 
> Apache APISIX will read from `/etc/resolv.conf`, if not, it will be a bug.

It's not a bug, maybe by design:  in apisix bin file, it's:
```
if not dns_resolver or #dns_resolver == 0 then
        local dns_addrs, err = local_dns_resolver("/etc/resolv.conf")
```
only when there is no dns_resolver, apisix will read /etc/resolv.conf. I think it's ok. For in-company using, just remove  dns_resolver conf, so it can read resolv.conf.